### DISCUSSION TITLE
[RELEASE] Facility [MAP UPDATE v1.02]

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#322] [Map Releases]
## This is a manual migration of forum user Naievil's map release `Facility`. Below contains the original post content. You can view the original post [here](https://nzportable.forumotion.com/t2003-release-facility-map-update-v1-02).

Hello everyone.

Thanks for taking the time to look into this map. I understand that the new Demo hasn't attracted as many map makers, so I wanted to start things off on a good note.

This map is named Facility. The general structure of the map is a remake of the Call of Duty: Black Ops (for Nintendo DS) map Facility. Both maps share a common structure; however, this is a remastered version that includes perks, more weapons, as well as dynamic shadows and the replacement and modification of some textures.

Here are some screenshots in Valve Hammer as well as some taken from in-game (low resolution warning):

![0](https://github.com/nzp-team/nzportable/assets/26030250/b9737d0d-b9e2-4c11-91ee-8c3d21e3bdd7)
![1](https://github.com/nzp-team/nzportable/assets/26030250/b9ced7c0-77a4-4997-aaae-2a1653a6f28a)

This map was designed to work on PSP. The entire idea of Nazi Zombies Portable relies on it being portable in the first place. Therefore, this map is partially trimmed (some objects from the original game removed, etc.) to allow smooth gameplay on the PSP. We are happy to say that this has been accomplished and the PSP (although sometimes lagging (especially in the first room. Just let it load and you will be fine)) can run smoothly when playing this map.

However, I'm resentful to say that this map does not have a Mystery Box. When choosing the guns located on the wall, I wanted to be firm but reasonable. Due to RAM limitations, the PSP Phat cannot handle loading every model (even when restricting the models in the Mystery Box) and then loading the map. I aimed for the ability for all users to play this map. If available, this feature may be added in the future.

This map features some textures that have been ripped and modified from the Call of Duty: Black Ops NDS version. The download links are provided free of charge, and no shortcut links that could generate revenue are used. The use of these textures is not affiliated with Nintendo and does not own the copyright to some of the textures in the map.
Likewise, my map as well features a model that was not previously given in the release of the NZ:P Demo. This model was released in the NZ:P Beta 1.1 files but was not included as a download. The provided download links do give users a copy of the model.
I have already spoken to an Administrator regarding both of these matters and have been given the go-ahead. However, if any Administrator, staff member, or (definitely) Game Developer has a problem with providing either one of these aspects, whether they be textures or models, they are more than willing to comment below as well as well as PM me for more details. 

Thanks for looking! Please read the README located inside of the downloaded folder and comment below on any bugs you encounter.

In addition, I'm looking to provide a "fly-trap" like Easter Egg in subsequent versions of this map. If anyone is willing to help me do so, please comment below with a solution that works in this new Demo (I have tested many, many options, all have failed).

The latest build now includes Double Tap!

__Changelog:__
* UPDATE v1.01: The free ammo bug has been fixed. Some textures have been changed into null to help map stability
* UPDATE v1.02: Now includes Double Tap!

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/12147853/Facility.zip)

**Comment by DiabloGane45** [comment]
This hasn't been commented on? This shant happen, it's a  fucking awesome map mate

**Comment by Jeyphr** [comment]
Yoooo! This looks sick! Can't wait to play!
